tests/status.py

```diff
@@ -72,6 +72,62 @@ def runTest(self):
         board.expect(prompt)
         self.result_message = 'ifconfig shows ip addresses: %s' % tmp
 
+class MacAddressCheck(rootfs_boot.RootFSBootTest):
+    '''Checked MAC addresses for all interfaces.'''
+    def __checkMac(self, mac_address, base_mac, mac_sequence):
+        result = int(base_mac, base=16) + mac_sequence
+        if mac_address:
+            mac_address = re.sub('[:]', '', mac_address.upper())[-6:]
+            if result == int(mac_address, base=16):
+                self.result_message = 'MAC addresses are sequential'
+            else:
+                raise ValueError('MAC addresses are not sequential')
+        else:
+            raise ValueError('Could not find appropriate MAC address')
+
+    def runTest(self):
+#       TODO: enable wi-fi mac address check from /sys/class once wi-fi tests are added
```

Comment:
Was saving the trouble of adding this entirely later, but if you say so, will remove these